### DISCUSSION TITLE
Mount directories via cirrus.yml

### DISCUSSION THREAD

**Discussion by josh-burton** [cirruslabs/tart#966] [Q&A]
The (limited) docs mention the best way to run scripts against a vm is via the cirrus cli, but I can't find any examples or docs on how to start a VM via cirrus and mount directories, as well as the other tart config options.

Can anyone help?

Also can cirrus be configured to start the vm and not exit once all tasks are complete?

**Comment by fkorotkov** [comment]
At the moment it's not possible to pass extra arguments via Cirrus CLI. It is possible to run tasks in ["dirty" mode](https://github.com/cirruslabs/cirrus-cli/tree/master?tab=readme-ov-file#running-cirrus-tasks) which will sync folder containing `.cirrus.yml` info the VM.

As for keeping a VM running after execution, there is no such option. What's your use case? Could you please add a bit more context why you need to keep VM running?
